Sources/Examples/EchoNIO/Generated/echo.pb.swift

```diff
@@ -0,0 +1,132 @@
+// DO NOT EDIT.
```

Comment:
Also symlink this, please (see the `.grpc.swift` file).

Reply:
Done